### DISCUSSION TITLE
Using Dataview Queries for Kanban List population

### DISCUSSION THREAD

**Discussion by Vi-Sri** [mgmeyers/obsidian-kanban#148] [Ideas & Feature Requests]
I was wondering if there could be a first class support to have dataview queries for list population. I tried and couldnt make it work. Any suggestions ?

**Comment by hepabolu** [comment]
+1

**Comment by barnabaslabs** [comment]
+1000

**Comment by Dreamsync** [comment]
Suuuuuper good idea.

**Comment by stoweboyd** [comment]
+1

**Comment by huyz** [comment]
See https://github.com/mgmeyers/obsidian-kanban/issues/345#issuecomment-1155314995 for progress

**Comment by hittjwX** [comment]
Any way to use the Kanban framework to visualize a data view of tasks would be a real joy. Seeing tasks as a table or in workflow columns is powerful. The Kanban columns could be set by a value like `workflow:: todo`, and then the note with Kanban would house the query and configuration for the order.
